soroban-builtin-sdk-macros/Cargo.toml

```diff
@@ -16,7 +16,7 @@ proc-macro = true
 syn = {version="2.0.39",features=["full"]}
 quote = "1.0.33"
 proc-macro2 = "1.0.69"
-itertools = "0.11.0"
+itertools = "0.10.5"
```

Comment:
this was downgraded because one of the new arkworks dependencies is locked into `itertools = "0.10.5"` so we had multiple versions. downgrading it seems fine since this is used for macro generation (if it didn't work we would've probably know during compile time).